app/uk/gov/hmrc/pillar2submissionapi/models/accountactivity/AccountActivitySuccessResponse.scala

```diff
@@ -15,13 +15,11 @@
  */
 
 package uk.gov.hmrc.pillar2submissionapi.models.accountactivity
-
-import play.api.libs.functional.syntax.given
-import play.api.libs.json.{Format, Json, __}
+import play.api.libs.json.{Format, Json}
 
 import java.time.{LocalDate, LocalDateTime}
 
-case class AccountActivitySuccessResponse(processingDate: LocalDateTime, transactions: Seq[AccountActivityTransaction])
+case class AccountActivitySuccessResponse(processingDate: LocalDateTime, transactionDetails: Seq[AccountActivityTransaction])
```

Comment:
In the last PR, I attempted to keep the serialised model a little bit separate from the in-memory model, just so I could use a nicer name. But it turns out we don't really operate on this much, so the benefits of a shorter name are nonexistent, and it came at the cost of having to fight swagger, so I switched back to a generic format.